app/src/main/java/com/vishaltelangre/nerdcalci/data/backup/BackupManager.kt

```
@@ -369,107 +397,192 @@ object BackupManager {
+                        var e = zipIn.nextEntry
+                        while (e != null) {
+                            if (!e.isDirectory && e.name.endsWith(Constants.EXPORT_FILE_EXTENSION)) {
+                                count++
+                            }
+                            zipIn.closeEntry()
+                            e = zipIn.nextEntry
+                        }
+                        count
+                    }
                 }
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)